packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1099,6 +1099,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
     const { disabled } = this.props;
 
     const handleInputKeyDown = (e: React.SyntheticEvent, searchInputProps: DropdownSearchInputProps) => {
+      e.stopPropagation();
```

Comment:
let's just stop propagation when the key was handled - for Tab, ArrowLeft, ArrowRight, Backspace

Reply:
Done